.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: python
+
+python:
+    - "2.7"
+
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get install -y openjdk-8-jre-headless
+
+install:
+    - wget https://wiki.dnanexus.com/images/files/dx-toolkit-current-ubuntu-14.04-amd64.tar.gz
+    - tar xf dx-toolkit-current-ubuntu-14.04-amd64.tar.gz 
+    - source dx-toolkit/environment
+    - wget https://github.com/dnanexus-rnd/dxWDL/releases/download/0.46/dxWDL-0.46.jar
+
+script:
+    - dx login --token "$DX_TOKEN" --noprojects && dx select "$DX_PROJECT"
+    - date_stamp="$(TZ=":US/Pacific" date +"%Y-%b-%d-%H-%M-%S")"
+    - dx mkdir "$date_stamp"
+    - workflow_id=$(java -jar dxWDL-0.46.jar compile smartseq2_single_sample/ss2_single_sample.wdl -destination "$DX_PROJECT":/"$date_stamp")
+    - >
+        analysis_id=$(dx run "$workflow_id" --yes --folder "$DX_PROJECT":/"$date_stamp" --brief -istage_0.fastq_read1=file-F7FxyyQ03bypfYvp1QPqXP7Y
+        -istage_0.fastq_read2=file-F7Fxyyj03byQ4F4z1Q7jkJ5F
```

Comment:
Where did all these files come from? How do they get into the environment where Travis is running the job? Is it possible to give them more human readable names? Ideally we would gsutil cp these down from a standard location to make it clearer where they came from, but some comments in the yaml about their origin might do for now.

Reply:
These are just the files from the demo that have been uploaded to DNAnexus. It's definitely very opaque now. I think ideally we'd stage these files on DNAnexus from the blue box as part of the test. For now I'll add some comments, though.